packages/salesforcedx-vscode-core/src/decorators/traceflag-time-decorator.ts

```diff
@@ -5,7 +5,10 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import * as moment from 'moment';
+import {
```

Comment:
The changes on this file control the formatting of time and date showed in the footer text added after running `SFDX: Turn On Apex Debug Log for Replay Debuger`
![replay_debugger_footer](https://user-images.githubusercontent.com/1041842/65006436-e854d800-d8b7-11e9-9c36-882052ea245f.png)